providers/hp/ilo/ilo.go

```diff
@@ -106,7 +106,7 @@ func (i *Ilo) CheckCredentials() (err error) {
 }
 
 // get calls a given json endpoint of the iLO and returns the data
-func (i *Ilo) get(endpoint string) (payload []byte, err error) {
+func (i *Ilo) get(endpoint string, useSession bool) (payload []byte, err error) {
```

Comment:
Hmm....these file changes arent asrockrack related, are they?

Reply:
yep, bad rebase, fixed in latest set of commits